src/app/[room]/page.tsx

```diff
@@ -0,0 +1,48 @@
+'use client'
+
+import { useEffect, useState } from 'react'
+import { socket } from '@/socket'
+import SocketIoTestSection from '@/components/socketIoTest/SocketIoTestSection'
+import StoryPoints from '@/components/StoryPoints'
+import CardsContainer from '@/components/CardsContainer'
+
+export type ChatEvent = {
+	message: string
+	timeStamp: string
+	room: string
+}
+
+export default function UserRooms({ params }: { params: { room: string } }) {
+	const [roomId, setRoomId] = useState('')
```

Comment:
todo: you don't need a roomId  as state, it already comes from the params :) You have other components with the same mistake, pls review them all :)

Reply:
DONE